src/main/java/com/google/sps/image/ImageScorer.java

```diff
@@ -35,6 +37,15 @@ public ImageScorer(String[] keywords) {
     this.keywords = keywords;
   }
 
+  private Map<String, Boolean> generateMap(List<String> elements) {
```

Comment:
This method should be static.

This tells the reader immediately that this method does not rely on the state, and is essentially called on classes and not on the instance of the class.

Reply:
Done.